tests/testsuite/owner.rs

```diff
@@ -183,10 +317,13 @@ fn simple_remove_with_asymmetric() {
 
     // The http_api server will check that the authorization is correct.
     // If the authorization was not sent then we would get an unauthorized error.
-    p.cargo("owner -r username")
-        .arg("-Zregistry-auth")
-        .replace_crates_io(registry.index_url())
-        .masquerade_as_nightly_cargo(&["registry-auth"])
-        .with_status(0)
-        .run();
+    let commands = ["-r", "--remove", "remove"];
+    for command in commands.iter() {
+        p.cargo(&format!("owner {} username", command))
+            .arg("-Zregistry-auth")
+            .replace_crates_io(registry.index_url())
+            .masquerade_as_nightly_cargo(&["registry-auth"])
+            .with_status(0)
+            .run();
+    }
```

Comment:
This removes in a loop, making the 2nd and 3rd iteration different than the first.  I'm not sure this is testing as much as dedicated tests.

Reply:
I'm guessing this test case is to test whether the authorization is sent or not. Using iteration should serve testing purposes, but I split it just to be on the safe side.

The way iterations are done in other test cases should be fine.